### PR TITLE
fix(twig): normalize twig templates, pt11 - INNO-1019

### PR DESCRIPTION
**Pull request by yhuard**
Updated:
- component-rss-link
- component-search-form
- component-site-header
- component-site-switcher

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 5a29a2de2b49395a74a7231913d6610e2b216be8

https://deploy-preview-665--europa-component-library.netlify.com

File: src/generic/generic-component-rss-link/generic-component-rss-link.twig [hunk 1]
```diff
@@ -1,11 +1,50 @@
...
+        ...
+      ]
+    - "system" (string) (default: "generic")
+#}
+
+{# Internal properties #}
+
+{% set _title = title|default('Get updates:') %}
+{% set _label = label|default('RSS') %}
+{% set _href = href|default('') %}
```
Comment (kalinchernev) [inline src/generic/generic-component-rss-link/generic-component-rss-link.twig]: I'd suggest having a default `#` just as here https://github.com/ec-europa/europa-component-library/pull/665/files#diff-18ac2dca89793da06a7a241c39610899R33

File: src/generic/generic-component-site-switcher/generic-component-site-switcher.twig [hunk 2]
```diff
@@ -1,50 +1,74 @@
...
-  - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+  Parameters:
+    - "variant" (string) (default: ''): can be '', 'header', 'footer'
+    - "political" (object):  {
+      - "href" (default: '')
+      - "label" (default: '')
+      - "is_active" (boolean) (default: false)
+    }
+    - "info" (object):  {
+      - "href" (default: '')
```
Comment (kalinchernev) [inline src/generic/generic-component-site-switcher/generic-component-site-switcher.twig]: There are defaults :slightly_smiling_face: 
- info: http://ec.europa.eu/info/index_en
- political: https://ec.europa.eu/commission/index_en

If not these, there's again room for a question if it's to be `#` or empty. (then please update https://github.com/ec-europa/europa-component-library/pull/665/files#diff-18ac2dca89793da06a7a241c39610899R33)

File: src/systems/ec/ec-preset/ec-preset-website/package.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
     "clean": "rimraf build dist",
     "dist": "rimraf dist && cross-env NODE_ENV=production run-p build:* -ln",
     "watch:scripts":
-      "chokidar \"../../../../generic/**/*.js\" \"../../**/*.js\" -i \"/node_modules|build|dist/\" -c \"npm run build:scripts\"",
```
Comment (kalinchernev) [inline src/systems/ec/ec-preset/ec-preset-website/package.json]: @raduchiriac :)

File: src/systems/ec/ec-template/ec-template-page/ec-template-page.twig [hunk 4]
```diff
@@ -22,7 +22,7 @@
       { href: "#", label: "Log in" },
     ],
   },
-  lanugages: languages
+  languages: languages
```
Comment (kalinchernev) [inline src/systems/ec/ec-template/ec-template-page/ec-template-page.twig]: :+1:

**Review by kalinchernev — COMMENTED** [review]
Only 1 general comment

File: src/generic/generic-component-site-switcher/generic-component-site-switcher.twig (hunk 2 above)
Comment (yhuard) [reply src/generic/generic-component-site-switcher/generic-component-site-switcher.twig]: those defaults link to the English version so if we choose to use them, we can set the labels as well

this is not something we have normalized yet: whether to use English defaults or empty defaults

@emeryro?

**Review by kalinchernev — APPROVED** [review]

**Pull request merged**